.github/workflows/linux.yml

```diff
@@ -398,9 +398,150 @@ jobs:
+              -DENABLE_CLDNN=OFF -DENABLE_INTEL_GPU=OFF -DENABLE_INTEL_NPU=OFF -DENABLE_HETERO=OFF -DENABLE_AUTO=OFF -DENABLE_AUTO_BATCH=OFF -DENABLE_MULTI=OFF -DENABLE_PROXY=OFF \
+              -DENABLE_FASTER_BUILD=y \
+              -DCPACK_GENERATOR=NPM \
+              -DENABLE_PYTHON=OFF \
+              -DENABLE_WHEEL=OFF \
+              -DCPACK_PACKAGE_FILE_NAME=genai_nodejs_bindings \
+              -S ./openvino -B ./build
+          cmake --build ./build --target package -j
+
+      - name: Run javascript tests
+        working-directory: ${{ env.GENAI_REPO }}/src/js
+        run: |
+          mkdir bin
+          cp ${{ env.BUILD_DIR }}/genai_nodejs_bindings.tar.gz bin
+          cd bin && tar -xzf genai_nodejs_bindings.tar.gz && rm genai_nodejs_bindings.tar.gz && cd ..
+          npm install
+          npm test
+
+      #
+      # Upload build artifacts and logs
+      #
+
+      - name: Upload genai nodejs bindings archive
+        if: ${{ always() }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: genai_nodejs_bindings
+          path: ${{ env.BUILD_DIR }}/genai_nodejs_bindings.tar.gz
+          if-no-files-found: 'error'
+
+  genai_nodejs_samples_tests:
+    name: NodeJS Samples Tests
+    needs: [ genai_nodejs_bindings ]
+    timeout-minutes: 30
+    defaults:
+      run:
+        shell: bash
+    runs-on: ubuntu-20.04-16-cores
+    env:
+      INSTALL_DIR: ${{ github.workspace }}/ov
+      GENAI_REPO: ${{ github.workspace }}/openvino.genai
+      MODELS_DIR: ${{ github.workspace }}/models
+      JS_SRC_DIR: ${{ github.workspace }}/openvino.genai/src/js
+      JS_SAMPLES_DIR: ${{ github.workspace }}/openvino.genai/samples/js/chat_sample
+      BIN_DIR: ${{ github.workspace }}/openvino.genai/src/js/bin
+    steps:
+      - name: Clone openvino.genai
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          path: ${{ env.GENAI_REPO }}
+          submodules: recursive
+
+      - name: Download genai nodejs bindings archive
+        uses: actions/download-artifact@v4
+        with:
+          name: genai_nodejs_bindings
+          path: ${{ env.BIN_DIR }}
+
+      - name: Unpack genai nodejs bindings archive
+        run: tar -xzf genai_nodejs_bindings.tar.gz && rm genai_nodejs_bindings.tar.gz
+        working-directory: ${{ env.BIN_DIR }}
+
+      - name: Install nodejs dependencies
+        run: npm install
+        working-directory: ${{ env.JS_SRC_DIR }}
+
+      - name: Download model for tests
+        run: npm run test_setup
+        working-directory: ${{ env.JS_SRC_DIR }}
+
+      - name: Install genai-node samples dependencies
+        run: npm install
+        working-directory: ${{ env.JS_SAMPLES_DIR }}
+
+      - name: Run tests
+        run: npm test
+        env:
+          MODEL_PATH: ${{ env.JS_SRC_DIR }}/tests/models/Llama-3.2-3B-Instruct-openvino-8bit
```

Comment:
it's very heavy model for tests. Please, use something like `katuni4ka/tiny-random-baichuan2`

Reply:
Agreed, replaced by [OpenVINO/Llama-3.1-8B-Instruct-FastDraft-150M-int8-ov](https://huggingface.co/OpenVINO/Llama-3.1-8B-Instruct-FastDraft-150M-int8-ov)